### PR TITLE
Removed hardcoded password

### PR DESCRIPTION
**Pull request by alichaddad**

### CODE REVIEW

File: templates/nodes/node.docker/actions.py [hunk 1]
```diff
@@ -89,9 +89,10 @@ def install(job):
 
     service.model.data.ipPrivate = ipaddress
     service.model.data.ipPublic = node.model.data.ipPublic
+    vag = service.aysrepo.servicesFind(actor='node.physical', name='vagrant4')[0]
     service.model.data.sshLogin = 'root'
-    service.model.data.sshPassword = 'gig1234'
-
+    service.model.data.sshPassword = vag.model.data.sshPassword
```
Comment (zaibon) [inline templates/nodes/node.docker/actions.py]: is there a default value for  `vag.model.data.sshPassword` ?

File: templates/nodes/node.docker/actions.py [hunk 2]
```diff
@@ -89,9 +89,10 @@ def install(job):
 
     service.model.data.ipPrivate = ipaddress
     service.model.data.ipPublic = node.model.data.ipPublic
+    vag = service.aysrepo.servicesFind(actor='node.physical', name='vagrant4')[0]
```
Comment (xmonader) [inline templates/nodes/node.docker/actions.py]: Why?

Comment (rkhamis) [inline templates/nodes/node.docker/actions.py]: Why indeed. Please remove.

File: templates/nodes/node.docker/actions.py [hunk 3]
```diff
@@ -89,9 +89,10 @@ def install(job):
...
     service.model.data.sshLogin = 'root'
-    service.model.data.sshPassword = 'gig1234'
-
+    service.model.data.sshPassword = vag.model.data.sshPassword
+    print(vag.model.data.sshPassword)
```
Comment (rkhamis) [inline templates/nodes/node.docker/actions.py]: Please remove print

**Pull request closed without merging**